src/Service/ConfigurationService.php

```diff
@@ -76,7 +75,7 @@ public function getApiKeyLive()
      */
     public function getEnvironment()
     {
-        return $this->systemConfigService->get(self::BUNDLE_NAME . '.config.environment');
+        return $this->systemConfigService->get(self::BUNDLE_NAME . '.config.environment') ? 'live' : 'test';
```

Comment:
can we call the library constants? \Adyen\Environment::LIVE and \Adyen\Environment::TEST?
it is unlikely to happen that we change the value of those, but we never know :)

Reply:
Done!